src/sbt-test/sbt-1.5/testkit/build.sbt

```diff
@@ -19,13 +19,15 @@ lazy val rules = projectMatrix
 
 lazy val input = projectMatrix
   .defaultAxes(VirtualAxis.jvm)
+  .jvmPlatform(scalaVersions = Seq(scala3Version))
   .jvmPlatform(
-    scalaVersions = rulesCrossVersions,
+    scalaVersions = Seq(V.scala213),
     settings = Seq(
-      scalacOptions += "-P:semanticdb:synthetics:on"
+      scalacOptions += "-Xsource:3"
```

Comment:
I think this shows how nice sbt-projectmatrix can be!

Reply:
I agree. The only thing I am worried about, is if we are the only project that use it. It introduces some new syntax to understand/learn, especially that sbt is usually not something that people master correctly.

Reply:
The rule authors shouldn't have to touch anything in projects bootstrapped by https://github.com/scalacenter/scalafix.g8/pull/19 unless they need to disable scala3 testing (which I enabled on purpose). Maybe that's something to discuss on the PR?

Reply:
or maybe they want to write only for scala 3. I think it should be easy to remove axes that the authors don't want?

Reply:
let's discuss this on the PR.